daemon/graphdriver/devmapper/deviceset.go

```diff
@@ -590,6 +590,9 @@ func (devices *DeviceSet) createFilesystem(info *devInfo) error {
 		devices.filesystem = determineDefaultFS()
 	}
 
+	logrus.Infof("devmapper: Creating filesystem %s on device(%s). START", devices.filesystem, info.Name())
+	defer logrus.Infof("devmapper: Creating filesystem %s on device(%s). END", devices.filesystem, info.Name())
```

Comment:
@estesp right and that's why we never claim that filesystem creation successfully finished. We just say START and END in an attempt to figure out when does operation finish (success/failure).

If personally feel that we are already doing this in many of the functions where we are logging START/END of function and END does not mean success. It just means that you returned from function.

So I feel this is good enough. But if you feel strongly about it, I can change it.